api/src/Data/Core/ContainerClients/RaceResultContainerClient.cs

```diff
@@ -15,13 +15,21 @@ public RaceResultContainerClient(ICosmosDbClient cosmosDbClient)
 
         public async Task<IEnumerable<RaceResult>> GetRaceResultsForMemberAsync(string memberId)
         {
-            return await GetRaceResultsForMembersAsync(new string[] { memberId });
+            return await GetRaceResultsForMembersAsync(new string[] { memberId }, null, null);
         }
 
         public async Task<IEnumerable<RaceResult>> GetRaceResultsForMembersAsync(IEnumerable<string> memberIds)
+        {
+            return await GetRaceResultsForMembersAsync(memberIds, null, null);
```

Comment:
Can we try to find a way to avoid the use of null if possible?

Reply:
What would you prefer here? I think this is an appropriate use

Reply:
I changed this to pass in `DateTime.MinValue` and `DateTime.MaxValue` instead